### ISSUE TITLE
Setup + Test WebOps Works

### ISSUE THREAD

**Issue by MaxLynam** [for-good/isgood-webapp#10]
Test dev/webops connect to update base project to amazee environments

- [x] dev enviro/s
- [x] prod enviro
- [ ] enviro variables & pre/post tasks (jason?)

Need to document #60 and also ensure that proper sync and updates occurs between developers, github and remote (amazee).

This needs to be a very solid and robust process and way of working, to ensure work is protected and we can stay in sync easily, and without error.

**Comment by MaxLynam** [comment]
We have been able to: 

*   sync 'enviro-dev' branch of webapp with the remote instance on amazee, for automated DevOps.
*   sync DB and files from remote back to local (different devs)
*   sync the repo and the configs and DB between devs
*   set keys for auth of different devs to be able to deploy and sync from remote.
*   started updating docs and instructions to ensure replicable, safe & secure process.

\_\_\_

\-> Requires further config of 'enviro-prod' via [https://docs.isgood.ai/welcome/contributing/drupal-lagoon-4devs/set-deploy-branch-and-remotes.](https://docs.isgood.ai/welcome/contributing/drupal-lagoon-4devs/set-deploy-branch-and-remotes.) .... do you reckon you'd be able to get this set up @Patchprogrammer ?  
  
Would like it set so each of the 'enviro-\*' branches can config for deploy to own instance, and then be able to sync back into the repo (& local) for further dev  (any way for automated sql & configs, without relying on humans remembering?).

**Comment by MaxLynam** [comment]
Have added in checklist item on this task for "enviro variables & pre/post tasks" for @Patchprogrammer ... as part of our workflow process (in [google working doc](https://docs.google.com/document/d/19gKolEHlUPfix_vSok7uLqVMur_vY_Bq0GQIRTgCcyg/), to then go to [gitbook](https://docs.isgood.ai/welcome/contributing/drupal-lagoon-4devs/set-deploy-branch-and-remotes) when sorted.

Very close now ... few minor bugs to resolve, then we go fast :)

**Comment by MaxLynam** [comment]
MAJOR achievement yesterday, as have proper sync between local-github-dev-github-local and back between developers ... what a painful and drawn out process it was to get this working properly !!!!

Next on checklist is to:
cleanup  a

- [ ]  little via gitignore (and test)
- [ ] try with deployed environments (ie. enviro-dev) for secure, managed and automated ci/cd.
- [ ] 

Excited that we got over this hurdle ... which was not expected to be an issue ... and, the over-rides and process we went through is working well during testing.
